external-crates/move/move-compiler/transactional-tests/tests/control_flow/if_branch_diverges_7.move

```diff
@@ -1,5 +1,7 @@
 //# run
 script {
+
+#[allow(dead_code)]
 fun main() {
```

Comment:
What code here is dead?

Reply:
This doesn't need it; I stuck it in as a catch to run another test. Thanks for the comment!